lib/action_subscriber/middleware/env.rb

```diff
@@ -46,14 +46,25 @@ def initialize(subscriber, encoded_payload, properties)
 
       def acknowledge
         fail ::RuntimeError, "you can't acknowledge messages under the polling API" unless @channel
+        return true if @has_been_acked
         acknowledge_multiple_messages = false
         @has_been_acked = true
         @channel.ack(@delivery_tag, acknowledge_multiple_messages)
         true
       end
 
+      def channel_open?
+        return false unless @channel
+        @channel.open?
+      end
+
+      def disable_safe_actions
```

Comment:
Is there a way to enable them?

Reply:
They’re enabled by default. But there isn’t a way to enable once you’ve disabled them if that’s what you mea.